src/CRM/CivixBundle/Resources/views/Code/test-phpunit.php.php

```diff
@@ -0,0 +1,42 @@
+<?php
+echo "<?php\n";
+?>
+
+// Allow autoloading of PHPUnit helper classes in this extension.
+$loader = new \Composer\Autoload\ClassLoader();
+$loader->add('CRM_', __DIR__);
+$loader->add('Civi\\', __DIR__);
+$loader->register();
+
+/**
+ * This is a generic test class for the extension, implementing PHPUnit tests.
+ *
+ * Testing functions must begin 'test' to be included in the PhpUnit tests.
+ */
+class <?php echo $testClass ?> extends \PHPUnit_Framework_TestCase {
+
+    /**
+     * Setup Method is executed before the test is executed (optional)
+     */
+    public function setUp() {
+        parent::setUp();
+    }
+
+    /**
+     * Tear Down Method is executed after the test was executed (optional)
+     * This can be used for cleanup
+     */
+    public function tearDown() {
+        parent::tearDown();
+    }
+
+    /**
+     * Example test case
+     *  Simple test if a boolean expression is true. This is implemented to show
+     *  an example for a test function
+     */
+    public function testExample() {
+        $myBool = True;
```

Comment:
Since this is going to be deleted anyway I'd suggest just limiting it to one line, also according to Civi coding standards the booleans should be in uppercase.

I know it's going to be changed anyway, but chances are new people seeing this test will copy it. Same goes for the assertion statement. There is no [right or wrong way](https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.static-vs-non-static-usage-of-assertion-methods) but personally I use `$this->assertTrue` more than static calls.